package.json

```diff
@@ -1,68 +1,68 @@
 {
-  "name": "actual",
-  "version": "0.0.1",
-  "private": true,
-  "description": "A local-first personal finance system",
-  "homepage": "https://github.com/actualbudget/actual/",
-  "bugs": {
-    "url": "https://github.com/actualbudget/actual/issues/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:actualbudget/actual.git"
-  },
-  "license": "MIT",
-  "workspaces": {
-    "packages": [
-      "packages/*"
+    "name": "actual",
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏would you mind reverting these formatting changes? I'm surprised our CI job isn't warning about this.

Reply:
Done